src/web/components/ShareIcons.tsx

```diff
@@ -33,6 +33,10 @@ const liStyles = (size: ShareIconSize) => css`
 	cursor: pointer;
 `;
 
+const hideSyles = css`
```

Comment:
I feel like the variable names doesn't match what you are trying to achieve. Maybe `topMarginStyles` might be better?

Reply:
Good shout

Reply:
Sorted :)